index.html

```diff
@@ -73,7 +73,298 @@ <h1>I'm a software developer</h1>
         </ul>
       </div>
       <!-- End of social media -->
+      <div class="button">
+        <a href="#">
+          <img src="images/scroll-button.png" alt="scroll-button" />
+        </a>
+      </div>
+    </section>
+    <!-- *start of work section -->
+    <section class="works">
+      <!-- h1, a line, 6 x cards, 
+        resume, languages, frameworks, skills  -->
+      <div>
+        <h1>My Recent Works</h1>
```

Comment:
- [ ] Good job identifying the titles and subtitles :+1:, but the title (e.g., "My Recent Works") doesn't have to use `<h1>` tag. You could use `<h2>` tag. This is essentially due to SEO reasons which many believe it is better to have only one `<h1>` tag on a webpage.